be/src/exprs/string_functions.cpp

```diff
@@ -4173,5 +4173,12 @@ StatusOr<ColumnPtr> StringFunctions::url_extract_parameter(starrocks::FunctionCo
         return url_extract_parameter_general(columns);
     }
 }
+// crc32
+DEFINE_UNARY_FN_WITH_IMPL(crc32Impl, str) {
```

Comment:
please add sql-tests for this new functions and make sure the crc32 will return the same result with other databases